package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "devDependencies": {
+    "block-publish": "^1.1.1",
```

Comment:
neat

next time we should put in in the @builder.io org and publish there, maybe people will use this and discover us!

Reply:
updated, going to make a little repo for it too